migrations/2025-06-14-141408_person_votes/up.sql

```diff
@@ -0,0 +1,61 @@
+ALTER TABLE person_actions
+    ADD COLUMN voted_at timestamptz,
+    ADD COLUMN upvotes bigint,
+    ADD COLUMN downvotes bigint;
```

Comment:
The timestamp doesnt seem so useful, anyway there would be no space to show it in UIs. And for the upvotes, downvotes columns int would be enough, no one is going to vote on a single user more than 2 billion times.